src/sig/qtesla/sig_qtesla.h

```diff
@@ -0,0 +1,48 @@
+#ifndef __OQS_SIG_QTESLA_H
+#define __OQS_SIG_QTESLA_H
+
+#include <oqs/oqs.h>
+
+#ifdef OQS_ENABLE_SIG_qTESLA_I
+
+#define OQS_SIG_qTESLA_I_length_public_key 1504
+#define OQS_SIG_qTESLA_I_length_secret_key 2112
+#define OQS_SIG_qTESLA_I_length_signature 1376
+
+OQS_SIG *OQS_SIG_qTESLA_I_new();
+
+extern OQS_STATUS OQS_SIG_qTESLA_I_keypair(uint8_t *public_key, uint8_t *secret_key);

```

Comment:
I don't think we need the `extern` keyword here or anywhere else in this file.

Reply:
This came from the nist-branch; should we remove it on both branches?

Reply:
There was a recent PR on nist-branch #387 that removed a bunch of superfluous extern's, but perhaps they missed some. Please remove it here, and I'll ask Ben to do the same on nist-branch.